src/SmartComponents/DriftPage/DriftTable/DriftTable.js

```diff
@@ -385,16 +380,20 @@ class DriftTable extends Component {
     render() {
         const { fullCompareData, filteredCompareData, systems, baselines, historicalProfiles, loading } = this.props;
 
+        this.masterList = this.formatEntities(
```

Comment:
Because these are already loaded as constants, you could just `this.masterList = this.formatEntities(systems, baselines, historicalProfiles);`

Reply:
Good catch, this led me to notice that I missed updating two method signatures as well! So I simplified those -- the call to renderTable and renderRows